app/templates/auth/create-user.html

```diff
@@ -0,0 +1,84 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Create account – Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+{% if not token %}
+<header class="page-heading-smaller">
+  <h1>Expired link</h1>
+</header>
+<p>
+    The link you used to create an account may have expired.<br/>
+    Check you’ve entered the correct link or <a href="{{ url_for('.create_buyer_account') }}">send a new one</a>.<br/>
+    If you still can’t create an account, email <a href="mailto:enquiries@digitalmarketplace.service.gov.uk">enquiries@digitalmarketplace.service.gov.uk</a>
+</p>
+{% else %}
+
+{%
+  with
+  smaller = true,
+  heading = "Create a new Digital Marketplace account",
+  context = email_address
+%}
+  {% include "toolkit/page-heading.html" %}
+{% endwith %}
+
+
+    <form autocomplete="off" action="{{ url_for('.submit_create_user', encoded_token=token) }}" method="POST" id="createUserForm">
+
+        <div class="grid-row">
+            <div class="column-two-thirds">
+                {{ form.hidden_tag() }}
+
+                {% if form.name.errors %}
+                <div class="validation-wrapper">
+                    {% endif %}
+                    <div class="question">
+                        {{ form.name.label(class="question-heading-with-hint") }}
+                        <p class="hint">
+                            Enter the name to be referred to on the Digital Marketplace
```

Comment:
Again, a lift from the supplier app to keep it consistent - the copy must have been OK'd at some point, but maybe we should take this opportunity to get it reviewed by @superroz or Sue for both apps.